text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,536 @@
+
+## Channel participation commands 
+In this section channel participation commands are described in a high level. The commands are presented in REST style
+but some details (i.e. exact formats, responses) are omitted for clarity.
+   
+### Joining a channel
+A channel is joined by providing the OSN with a config block:
+
+  ```Join channel-name channel-config-block```
+  
+  or
+  
+  `POST /channels/channel-name` with body containing `channel-config-block`
+  
+The body contains a config block that contains all the details about the channel. 
+
+If the OSN is joining a new channel, the config block is the genesis block of the channel. It looks like the first 
+block of an application channel that is generated by the OSN, when a channel creation transaction is submitted to the 
+system channel.
+
+If the OSN is joining an existing channel, the config block is simply the last config block of the channel, retrieved 
+from one of the other OSNs.
+
+If the channel already exists, the call will fail.
+
+If the system channel exists, the call will fail. 
+          
+The OSN receiving this call will inspect the block and conduct some sanity checks in order to make sure that the block 
+is well-formed.
+
+The OSN will try to find itself in the consenters set, and determine if it is a follower or a member of the cluster
+executing consensus. 
+
+#### Definition of cluster member and cluster follower
+* A cluster **follower** is an OSN that is not defined in the channel's consenters set and does not take part in 
+  consensus. It just pulls blocks from other OSNs (which are members of the cluster).  
+* A cluster **member** OSN is defined in the channel's consenters set and takes part in consensus.
+
+Configuration changes can cause an OSN to switch between the follower and member roles. 
+ 
+* A follower evaluates incoming config blocks and checks whether it was added to the consenters set. If it was added, 
+  it starts the runtime components that execute consensus, thus becoming a member the cluster.
+* A member evaluates incoming config blocks and checks whether it was removed from the consenters set. If it was 
+  removed, it stops the runtime components that execute consensus, but continues to operate as a cluster follower.
+     
+An OSN operating as a cluster follower is a new feature introduced by this work.
+
+#### Two ways to join     
+When an OSN joins a channel as a **follower**, it start to pull blocks from other OSNs. It will continue
+to do so until it **catches up** with the config block received in the invocation. From then on, it will continue to pull 
+blocks, but will inspect each incoming config block and check whether it is included in the consenters set. 
+When channel admins add it to the consenters set, it will detect that and start the runtime components that execute 
+consensus, thus becoming a member the cluster. 
+
+When an OSN joins a channel as a **member**, it first pulls blocks from other OSNs util it **catches
```

Comment:
Very minor typo -- "util" instead of "until"

Reply:
Fixed.